Makefile

```diff
@@ -46,20 +45,21 @@ test:
 	@$(MOCHA) \
 		--timeout 120s \
 		--slow 3s \
-		--grep "$(FILTER)" \
 		--bail \
 		--reporter spec \
 		--compilers js:babel-register \
 		test/
 
-test-all:
+test-watch:
 	@$(MOCHA) \
 		--timeout 120s \
 		--slow 3s \
 		--bail \
 		--reporter spec \
 		--compilers js:babel-register \
-		test
+		--watch \
+		--grep "$(filter-out $@,$(MAKECMDGOALS))" \
```

Comment:
This is not a good pattern, which was why I introduced the `FILTER` variable previously. Please see http://stackoverflow.com/a/6273809.